testground/benchmark/benchmark/sendtx.py

```diff
@@ -78,16 +81,43 @@ def generate_load(num_accounts, num_txs, global_seq, **kwargs):
                 print("test task failed", e)
 
 
-def prepare_txs(global_seq, num_accounts, num_txs):
-    accounts = [gen_account(global_seq, i + 1) for i in range(num_accounts)]
-    txs = []
-    for i in range(num_txs):
-        for acct in accounts:
-            txs.append(acct.sign_transaction(test_tx(i)).rawTransaction.hex())
-            if len(txs) % 1000 == 0:
-                print("prepared", len(txs), "txs")
+async def async_write_tx(file_path, raw_tx):
+    async with aiofiles.open(file_path, "w") as file:
+        await file.write(ujson.dumps({"rawTransaction": raw_tx.hex()}))
+
+
+def get_txs_foler(outdir: Path):
+    folder = outdir / TMP_TXS_FOLDER
+    folder.mkdir(parents=True, exist_ok=True)
+    return folder
+
 
-    return txs
+async def prepare_txs(outdir, global_seq, num_accounts, num_txs, max_workers=8000):
```

Comment:
How does asyncio help here?

Reply:
I suggest to keep it simple if not certain, for example, keep the `prepare_txs` as is, and in the `gen_txs` command, simply write the output to the files.

Reply:
you can make `prepare_txs` parallel in a separate PR

Reply:
do you mean generate one txs file instead of tx files by index?

Reply:
one file per node, named after `global_seq`, should be good enough?